controllers/datamover.go

```diff
@@ -250,13 +250,14 @@ func (r *DPAReconciler) createResticSecretsPerBSL(dpa *oadpv1alpha1.DataProtecti
 				r.Log.Info(fmt.Sprintf("Error parsing provider secret %s for backupstoragelocation", secretName))
 				return nil, err
 			}
-			repobase := "s3:s3.amazonaws.com"
-			if bsl.Spec.Config["s3Url"] != "" {
+			repobase := "s3.amazonaws.com"
 
+			if bsl.Spec.Config["s3Url"] != "" {
 				repobase = bsl.Spec.Config["s3Url"]
+				repobase = strings.TrimPrefix(repobase, "s3:")
 			}
 			repobase = strings.TrimSuffix(repobase, "/")
-			repo := repobase + "/" + bsl.Spec.ObjectStorage.Bucket
+			repo := "s3:" + repobase + "/" + bsl.Spec.ObjectStorage.Bucket
```

Comment:
@savitharaghunathan Do we have to add the `s3` prefix ? Why are we not directly using the `s3Url` specified under BSL config ? I believe this might work for noobaa case but might not work for other s3 compatible storage services like minio.

Reply:
`s3Url` from bsl config caused VolSync to error. Hence adding `s3:` prefix.

Reply:
@eemcmullan Thanks for the info https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#minio-server
Seems this is the expected format ! Approving the PR !

Reply:
Thank you @shubham-pampattiwar @eemcmullan